GlacierProtocol/glacierprotocol.github.io
# New markdown-based site with support for PDF

This change:
* Updates the Glacier site to be markdown-based
* Allows generating a PDF from the same source. This PDF is to be packed together with the zip package of the Glacier protocol.
* The site continues being managed and hosted using Github Pages, but you can now test things locally with Docker instead of having to install Jekyll.

## How to test this

There's a `make` file with targets that allow you to test this locally:

1. Make sure you have Docker installed
2. Clone this fork and `cd` into it
2. `make site` starts a local deployment of the website at `http://localhost:4000`. This should match exactly what you'll get once you host this site with Github Pages
3. `make pdf` generates a new pdf based on the markdown source.
